Manual/GAP_code/vecspaces_example/complete_example.g

```diff
@@ -35,27 +35,39 @@ BindGlobal( "SQVec", CreateCapCategoryWithDataTypes(
 
 SetIsAbelianCategory( SQVec, true );
 
+SQVec!.category_as_first_argument := true;
+
 BindGlobal( "QQ", HomalgFieldOfRationals( ) );
 
 ##
 InstallMethod( SQVecObj,
-               [ IsInt ],
+               [ IsSQVecCat, IsInt ],
                
-  function( dim )
+  function( cat, dim )
```

Comment:
Since this is only a demo, I suggest not making `cat` a variable here and just keeping the hard-coded reference to `SQVec`. Of course this is not how we would suggest organizing the code in more complicated examples, but we would also not recommend creating a global category `SQVec`, so.... :D

Reply:
Done. I had the same inner conflict. ^^